imageroot/systemd/user/mariadb-app.service

```diff
@@ -6,6 +6,8 @@ After=mariadb.service phpmyadmin-app.service
 [Service]
 Environment=PODMAN_SYSTEMD_UNIT=%n
 EnvironmentFile=%S/state/environment
+EnvironmentFile=%S/state/secrets/root.secret
+EnvironmentFile=%S/state/secrets/phpmyadmin.secret
```

Comment:
I like the sane idea to not store secrets in the state/environment file!

However the two files could be squashed into one as they are in a shell-like syntax.

Reply:
done